simple-wp-helpdesk/assets/src/portal-v2/style.scss

```
@@ -0,0 +1,207 @@
+
+.swh-msg {
+	max-width: 75%;
+	&-customer { align-self: flex-end; }
+	&-staff    { align-self: flex-start; }
+}
+.swh-msg-bubble {
+	padding: 10px 14px;
+	border-radius: var(--swh-radius-md, 8px);
+	font-size: 14px;
+	line-height: 1.5;
+	word-wrap: break-word;
```

Stylelint (17.11.1)

[error] Expected "word-wrap" to be "overflow-wrap" (property-no-deprecated)

(property-no-deprecated)